gordo_components/cli/cli.py

```diff
@@ -93,7 +93,8 @@ def gordo():
 @click.option(
     "--evaluation-config",
     envvar="EVALUATION_CONFIG",
-    default='{"cv_mode": "full_build"}',
+    default='{"cv_mode": "full_build", '
+    '"scoring_scaler": "sklearn.preprocessing.RobustScaler"}',
```

Comment:
Just noticing this now, but should be rather do:
```python
yaml.safe_dump({"cv_mode": "full_build", "scoring_scaler": "sklearn.preprocessing.RobustScaler"})
```
instead of writing it out as a string, which is less likely to catch syntax errors?